docs/_sass/_main.scss

```diff
@@ -182,6 +183,180 @@ button {
     align-content: center;
 }
 
+.logo {
+    width: 80%;
+}
+
+.search-icon {
+    margin: auto 0px;
+}
+
+.flex {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -moz-flex;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;
+    flex-flow: row wrap;
+    align-content: space-between;
+}
+
+#sidebar-search {
+    background-color: $color-appBG;
+    width: 375px;
+    height: 100vh;
+    position: fixed;
+    display: block;
+    top: 0;
+    right: 0;
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
+    #sidebar-search {
+        width: 100%;
+    }
+}
+
+.searchbar-title-wrapper {
+    padding: 20px;
+}
+
+.search-title {
+    font-size: 17px;
+    padding-bottom: 20px;
+}
+
+#toggle-search-close {
+    margin: auto;
+    margin-left: 0px;
+    margin-right: 10px;
+}
+
+/* Sidebar Layer */
+#sidebar-layer {
+    position: fixed;
+    /* Sit on top of the page content */
+    display: none;
+    /* Hidden by default */
+    width: 100%;
+    /* Full width (cover the whole page) */
+    height: 100%;
+    /* Full height (cover the whole page) */
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+    z-index: 2;
+    /* Specify a stack order in case you're using a different order for other elements */
+}
+
+/* All gsc id & class are Google Search related */
```

Comment:
This is still not specific at all though. What exactly is this element? is it an input? is it a button? something else?

Reply:
Noted, the recent one is already updated @shawnborton. Please let me know what you think of it!